api/interservice/compliance/reporting/reporting.proto

```diff
@@ -428,3 +428,10 @@ message AssetListRequest {
 message AssetListResponse {
 	repeated Assets assets = 1;
 }
+message ComplianceConfig {
+    string policy_name = 1;
```

Comment:
We can utilise it here so that we can call the same service for every policy updates